src/go/rpk/pkg/cli/profile/create.go

```diff
@@ -311,6 +313,33 @@ var ErrNoCloudClusters = errors.New("no clusters found")
 // it automatically. This returns ErrNoCloudClusters if the user has no cloud
 // clusters.
 func PromptCloudClusterProfile(ctx context.Context, cl *cloudapi.Client) (p config.RpkProfile, requiresMTLS, requiresSASL bool, err error) {
+	// We get the org name asynchronously because this is a slow request --
+	// doing it while the user does a slow operation of selecting a cluster
+	// hides the delay.
+	//
+	// We also get namespaces immediately for two reasons: (1) we always
+	// want the namespace name, and (2) we might need a list of all
+	// namespaces.
+	var (
+		org     cloudapi.Organization
+		orgErr  error
+		orgDone = make(chan struct{})
+
+		nss     []cloudapi.Namespace
+		nssErr  error
+		nssDone = make(chan struct{})
+	)
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	go func() {
+		defer close(orgDone)
+		org, err = cl.Organization(ctx)
```

Comment:
Should this be orgErr ? and below nssErr ?

Reply:
good catch